interface/billing/billing_report.php

```diff
@@ -411,927 +411,957 @@ function MarkAsCleared(Type) {
     document.onclick = TakeActionOnHide;
 </script>
 <!-- =============Included for Insurance ajax criteria==== -->
+<?php
+
+//to determine and set the form to open in the desired state - expanded or centered, any selection the user makes will become the user-specific default for that page
+// collectAndOrganizeExpandSetting() contains a single array as an argument, containing one or more elements, the name of the current file is the first element, if there are linked 
+// files they should be listed thereafter, please add _xpd suffix to the file name
+
+$arr_files_php = array("billing_report_xpd");
+
+// needed for the jQuery part of script
+if (count($arr_files_php) > 1) {
+    $arr_files_string = implode(",", $arr_files_php);
+} else {
+    $arr_files_string = $arr_files_php[0];
+}
```

Comment:
I think the block above can just go into the expand_contract_js.php script.

Reply:
(btw, we are getting close to being done)
If that is true, then you can put the $current_state = collectAndOrganizeExpandSetting($arr_files_php); line into the require_once("$srcdir/expand_contract_inc.php"); script.
Then you will just have:
```
$arr_files_php = array("billing_report_xpd");
require_once("$srcdir/expand_contract_inc.php");
```

Reply:
:)